### PR TITLE
Replace Sensio ParamConverter by ValueResolver

### PR DESCRIPTION
**Pull request by dougetovski**
- Replace ParamConverter by ValueResolver
- Removing depedency with sensio-framework-extra-bundle

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,16 +23,15 @@
     "conserto/pomm-symfony-bridge":  "^4.0",
     "symfony/framework-bundle": "~5.0|~6.0",
     "twig/twig": "~3.0",
-    "sensio/framework-extra-bundle": "~5.0|~6.0"
+    "symfony/http-kernel": "^6.2"
```
Comment (neufman) [inline composer.json:26]: A voir pour conserver la compatibilité sf 5

Comment (dougetovski) [reply composer.json:26]: good point. I've changed the target branch to 5.0, because there is some breaking changes (Removal of ParamConverter)

**Review by neufman — DISMISSED** [review]

File: sources/lib/ValueResolver/EntityValueResolver.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        try {
+            $entity = $model->findByPk($this->getPk($model, $request));
+        } catch (\LogicException $e) {
+            if ($options["optional"] === false) {
+                throw $e;
+            }
+        }
+
+        return [$entity];
```
Comment (maximechagnolleau) [inline sources/lib/ValueResolver/EntityValueResolver.php:43]: yield $entity;

File: sources/lib/ValueResolver/EntityValueResolver.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            !is_a($argumentType, FlexibleEntityInterface::class, true)
+        ) {
+            return [];
+        }
+
+        $options = $this->getOptions($argument);
+
+        $model = $options['session']->getModel($options['model']);
+
+        return [$model->findByPk($this->getPk($model, $request))];
```
Comment (dougetovski) [inline sources/lib/ValueResolver/EntityValueResolver.php]:
```suggestion
        yield $model->findByPk($this->getPk($model, $request));
```

**Review by maximechagnolleau — DISMISSED** [review]

**Pull request merged**